dbt/adapters/bigquery/relation.py

```diff
@@ -16,6 +17,14 @@
 class BigQueryRelation(BaseRelation):
     quote_character: str = "`"
     location: Optional[str] = None
+    # why do we need to use default_factory here but we can assign it directly in dbt-postgres?
```

Comment:
Does anyone know why I needed to use field(default_factory...) here when `dbt-postgres` allows me to register this with a simple list? Removing this causes unit tests to fail due to mashumaro errors for using a mutable default.

Reply:
Figured it out, we don't test for this in `dbt-postgres`. This does not work with `.validate()` unless we use `default_factory`.